### DISCUSSION TITLE
For 1.19.4 start upgrading Holograms to the new "Display" Minecraft settings

### DISCUSSION THREAD

**Discussion by TomLewis** [Zrips/CMI#7840] [Enhancement]
https://youtu.be/gf5YNRAoTcM?t=313
https://minecraft.fandom.com/wiki/Display

You should be able to easily transiton to the new method of Displays that do not use armor stands for 1.19.4+

**Comment by Zrips** [comment]
Chat Bubbles currently uses this new system which was basically a test of a concept and ground preparation for this new handling. Its underway

**Comment by MrCrazys** [comment]
> https://youtu.be/gf5YNRAoTcM?t=313 https://minecraft.fandom.com/wiki/Display
> 
> You should be able to easily transiton to the new method of Displays that do not use armor stands for 1.19.4+

I would love to see this, however I also think there should be an option to use armor stands still.

**Comment by Ste3et** [comment]
Zrips can you add TextDisplays as Hologram option too?
I would love to see these feature in cmi, i have so mutch ideas what i can do with it.
